Litmus-Leader.md

```diff
@@ -0,0 +1,29 @@
+# Litmus Leader Program
+The Litmus Leader Program aims to identify community contributors & leaders in the Litmus community and reward them for their valuable contribution to Cloud-Native world by contributing to the LitmusChaos project.
+This program is the a Rewards and Recognitions campaign for the LitmusChaos project to award our most prominent contributors in various fields like development, advocacy, writing, content creation, etc. and be an example for others.
+
+----------------------
+
+## Leadway to Litmus Leader 💥:
+
+1. Active Participation: We encourage you to be participative in feedback and attending our Monthly Community Meetups, GitHub LitmusChaos discussions, and on other handles as well (Twitter, LinkedIn, Slack, etc.)
```

Comment:
Can we separate these points? Feedback on slack/github and attendance on community sync-ups is deemed enough for active project participation IMO. Participation on social media in more of advocacy.